### DISCUSSION TITLE
How to fetch the deleted data before saving to the database

### DISCUSSION THREAD

**Discussion by Rahul-codoffer** [filamentphp/filament#10775] [Help] [answered]
When i fetch the data from db in beforeSave function , then i didn't get the deleted data .
I have used the repeater and before saving to the data base i want the id of deleted record.

**Comment by leandrocfe** [comment]
Could you explain more what you need please? Share some code you are trying

**Comment by Rahul-codoffer** [comment]
i was using the filament repeater and i want the data which is getting delete and want to fire the job 
but for now i have used the relation manager and soft delete so get the which records are deleted.
in before Save function i got the only data which is going stored in db .
but i want which records are deleted in repeater

**Comment by leandrocfe** [comment, marked as answer]
Maybe using an observer to compare the current/changed data?
https://github.com/chelout/laravel-relationship-events

**Comment by Rahul-codoffer** [comment]
Thanks buddy !
